authorizations/authorization-ranger/src/test/java/org/apache/gravitino/authorization/ranger/integration/test/RangerITEnv.java

```diff
@@ -106,11 +108,30 @@ public static void cleanup() {
     }
   }
 
+  static void startHiveRangerContainer() {
+    containerSuite.startHiveRangerContainer(
+        new HashMap<>(
+            ImmutableMap.of(
```

Comment:
Why do we need a nested Map?

Reply:
Yes, you are right, I already fixed it in the next PR.